ouroboros-consensus/docs/Onboarding.md

```diff
@@ -0,0 +1,144 @@
+# Onboarding to the Consensus Team
+
+The content in this file should help orient a new contributor to the Consensus
+Layer: it's fundamental purpose and its neighboring components.
+
+Please open PRs to add particularly useful content here. Though we would like
+the file to remain as approachable as possible, therefore small and
+narrowly-scoped.
+
+## Very High-Level Motivation
+
+At a very high level, a net of Cardano nodes should exhibit the following
+concrete behaviors.
+
+  * The net should diffuse transactions throughout the net as rapidly as
+    possible.
+
+  * The net should diffuse blocks through the net as rapidly as possible.
+
+  * Whenever the ledger rules specify that (a stake pool operating) a particular
```

Comment:
Not the ledger rule: the consensus algorithm.

Reply:
Yep.